code/drasil-lang/Language/Drasil/ExprExtract.hs

```diff
@@ -0,0 +1,50 @@
+module Language.Drasil.ExprExtract (egetDoc)where
```

Comment:
This is the same as above. This one is correct, the previous one shouldn't be merged in (wrong name).

Reply:
Changes were made in 8c1738588678f66ed29a6da31c4b7a6f243eaabe.